src/main/java/core/basesyntax/service/ReadFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ReadFromFile {
+    List<String> readFromFile();
```

Comment:
```suggestion
    List<String> readFromFile(String pathToFile);
```

Reply:
Done.
Replace 
private static final String dayReportFilePath = "src/main/resources/transaction.csv";
to Main class